bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -11,13 +11,13 @@
 from os import environ
 
 # Prevent cppyy's check for the PCH
-environ['CLING_STANDARD_PCH'] = 'none'
+environ["CLING_STANDARD_PCH"] = "none"
```

Comment:
Sorry to be picky, but could we maybe have a separate commit for non-functional changes such as these ones?

Reply:
Yes there is, the first commit is only style changes and the second one the functional changes